code/index.html

```diff
@@ -1,22 +1,146 @@
+           alt="image city silhouette"
+           />          
+           <h2>China has a huge wealth-gap problem — and it's getting worse</h2>
+         </a>         
+        </div>
+      </div>
+      <div class="news">
+        <div class="card"> 
+          <a href="#">
+           <img 
+           src="https://images.unsplash.com/photo-1562596127-3885a12dcdb6?ixid=MXwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHw%3D&ixlib=rb-1.2.1&auto=format&fit=crop&w=1050&q=80" 
+           alt="math studies"
+           />
+           <h2>The rich are minting money in the pandemic like never before</h2>
+          </a>
+       </div>
+      </div> 
+      <div class="news">
+        <div class="card"> 
+         <a href="#">         
+             <img 
+             src="https://images.unsplash.com/photo-1610375461246-83df859d849d?ixid=MXwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHw%3D&ixlib=rb-1.2.1&auto=format&fit=crop&w=1050&q=80" 
+             alt="image gold bars"
+             />
+           <h2>Gold rises from december low amid ‘Broad Risk-Off Sentiment’</h2>
+          </a>
+        </div>
+      </div> 
+      <div class="news">
+        <div class="card"> 
+          <a href="#">
+           <img 
+           src="https://images.unsplash.com/photo-1550565118-3a14e8d0386f?ixid=MXwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHw%3D&ixlib=rb-1.2.1&auto=format&fit=crop&w=1050&q=80"
+           alt="image bills and coins"
+           />
+           <h2>China officially backs a crypto currency and establishes it as their official coin</h2>
+          </a> 
+        </div>
+      </div>  
+      <div class="news">
+        <div class="card">
+          <a href="#">
+            <img 
+           src="https://images.unsplash.com/photo-1559526324-593bc073d938?ixid=MXwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHw%3D&ixlib=rb-1.2.1&auto=format&fit=crop&w=1050&q=80" 
+           alt="image phone screen stock market"
+           />            
+           <h2>Italian bond yields creep up before key votes in parliament</h2>
+          </a>        
+        </div>
+      </div>
+      <div class="news">
+       <div class="card"> 
+          <a href="#">
+            <img 
+             src="https://images.unsplash.com/photo-1546450334-5a84ac3a1f0e?ixid=MXwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHw%3D&ixlib=rb-1.2.1&auto=format&fit=crop&w=1053&q=80" 
+             alt="picture prison fence"
+           />
+            <h2>Samsung's Lee receives two-and-a-half year jail term from Seoul court</h2>
+          </a> 
+        </div>
+      </div> 
     </section>
 
-    <section class="">
-      <!-- The other news  -->
-    </section>
+    <!--***Animation bottom news-letter***-->
+    <container class="animation">
+      <a class="news-letter" href="#"><h3>Don't forget to sign up for our news-letter!</h3></a>
+    </container> 
+    
+   <!--***Footer back to top button***-->
+    <footer>
+      <button id="btn"><a href="#top">Back to Top</a></button>
+    </footer>
```

Comment:
Nice with a footer! Instead of having a link inside the button I would suggest that you either style the link as a buttonor add an onclick attribute to the button with the link as target